dev_cycle/terraform/input.tf

```diff
@@ -6,3 +6,4 @@ variable "key_pair_name" {}
 variable "machine_name" {}
 variable "vpc_id" {}
 variable "aws_iam_role_id" {}
+variable "ebs_kms_key" {}
```

Comment:
I would prefer we pass in the the ARN here, so renaming it to `ebs_kms_key_arn` is preferred.

Reply:
changed to arn